drivers/gpu/drm/i915/vlv_dsi.c

```diff
@@ -1674,6 +1680,7 @@ static const struct drm_connector_helper_funcs intel_dsi_connector_helper_funcs
 };
 
 static const struct drm_connector_funcs intel_dsi_connector_funcs = {
+	.detect = intel_dsi_detect,
 	.late_register = intel_connector_register,
```

Comment:
It is unnecessary to add the "detect" callback for dsi connector as it returns the status of "connector_status_connected" when calling the function of drm_helper_probe_detect.

Reply:
Yes, drm_helper_probe_detect hase this logic, but intel_initial_modeset.c use  connector->funcs->detect instead of drm_helper_probe_detect,   this will  bypass MIPI DSI connector detection due to the lack of detect function.